addons/calendar/static/src/views/fields/many2many_attendee_expandable.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="calendar.Many2ManyAttendeeExpandable" t-inherit="calendar.Many2ManyAttendee" t-inherit-mode="primary" owl="1">
+        <xpath expr="//TagsList" position="attributes">
+            <attribute name="itemsVisible">state.expanded ? tags.length : 5</attribute>
+        </xpath>
+        <xpath expr="//TagsList" position="before">
+            <t t-if="tags.length > 5">
+                <div class="d-flex w-100">
+                    <div class="flex-grow-1">
+                        <strong><t t-out="tags.length"/> attendees</strong>
+                        <t t-set="partners" t-value="props.record.preloadedData.partner_ids"/>
+                        <br/><t t-out="partners.filter(p => p.status === 'accepted').length"/> accepted
+                        <br/><t t-out="partners.filter(p => p.status === 'declined').length"/> declined
+                        <br/><t t-out="partners.filter(p => p.status !== 'accepted' &amp;&amp; p.status !== 'declined').length"/> uncertain
```

Comment:
Could be replaced by getters in the component level. For example

```js
export class Many2ManyAttendeeExpandable extends Many2ManyAttendee {
   get partners() {
        return this.props.record.preloadedData.partner_ids;
    }

    get acceptedPartnersCount() {
        return this.partners.filter(p => p.status === 'accepted').length;
    }
    ...
}
```
and in the template you just use 

```xml
    <t t-out="acceptedPartnersCount"/> accepted
```